packages/storyshots/.jestrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
The second .jestrc file is for the `test-examples` script. Running `yarn test` uses the .jestrc in the root directory, but `yarn test-examples` uses lerna to run npm test in each package's directory. If I used `"test": "jest --config ../../.jestrc"` in the storyshots directory to reference the jestrc in the root, I ran into this error:
```
lerna ERR! test Errored while running script in 'storyshots'
lerna ERR! execute Error: Command failed: npm run test
lerna ERR! execute ● Validation Error:
lerna ERR! execute
lerna ERR! execute   Module ./node_modules/jest-enzyme/lib/index.js in the setupTestFrameworkScriptFile option was not found.
lerna ERR! execute
lerna ERR! execute   Configuration Documentation:
lerna ERR! execute   https://facebook.github.io/jest/docs/configuration.html
```
The `rootDir` is also different for `yarn test` and `yarn test-examples`, so I copied the .jestrc from the root dir here and made a few minor changes to the paths so I could do `"test": "jest --config ./.jestrc"` in the storyshots package.json.